panel/_styles/json.css

```diff
@@ -0,0 +1,194 @@
+.json-formatter-row {
```

Comment:
Unless this is intentionally added, it should be possible to include styles from an npm package.

Reply:
What's the syntax for that?

Reply:
That's a good question, as I didn't test this outside bokehjs. I will get back to you on this later (so perhaps in another PR, if this one progresses quickly).

Reply:
As far as I can tell they don't actually ship the CSS with the npm bundle anyway.